modules/create_timepoint/jsx/createTimepointIndex.js

```diff
@@ -150,7 +150,7 @@ class CreateTimepoint extends React.Component {
             this.handleVisitLabel();
           }
           // Populate the select options for languages.
-          if (data.languages) {
+          if (Object.keys(data.languages).length > 1) {
             state.form.options.languages = data.languages;
             state.form.display.languages = true;
```

Comment:
Hi @CamilleBeau you could just modify line 155 to be:
`state.form.display.languages = Object.keys(data.languages).length > 1;`
then the form options on line 154 would still contain the data.languages if it's ever needed when there's only 1 option available and while languages on the form are still being hidden to the user.

Reply:
Done!